wireworld.py

```diff
@@ -0,0 +1,154 @@
+import json
+import sys
+import copy
+
+import numpy as np
+
+initial_state = [
+[0, 0, 0, 3, 0, 0, 0], 
+[0, 3, 0, 3, 0, 3, 0], 
+[3, 0, 3, 0, 3, 0, 3], 
+[1, 0, 0, 3, 0, 0, 1], 
+[2, 0, 3, 3, 3, 0, 2], 
+[0, 3, 0, 3, 0, 3, 0], 
+[0, 0, 0, 3, 0, 0, 0]
+]
+
+
+def config(initial_state):
```

Comment:
config looks good! As a bit of extra credit, you could add some more validation in the form of a `try except` to catch non existing file names!